--------------------Moonraker Log Start--------------------
data_path: /home/pi/printer_data
is_default_data_path: False
config_file: /home/pi/klipper_config/moonraker.conf
startup_warnings: []
verbose: False
debug: False
asyncio_debug: False
is_backup_config: False
log_file: /home/pi/klipper_logs/moonraker.log
software_version: v0.7.1-771-ge628b24
python_version: 3.7.3 (default, Jan 22 2021, 20:04:44)  [GCC 8.3.0]

#################### Moonraker Configuration ####################

[server]
host = 0.0.0.0
port = 7125
max_upload_size = 1024

[file_manager]
enable_object_processing = False

[authorization]
cors_domains = 
	https://my.mainsail.xyz
	http://my.mainsail.xyz
	http://*.local
	http://*.lan
trusted_clients = 
	10.0.0.0/8
	127.0.0.0/8
	169.254.0.0/16
	172.16.0.0/12
	192.168.0.0/16
	FE80::/10
	::1/128

[octoprint_compat]

[history]

[update_manager]
refresh_interval = 168

[update_manager mainsail]
type = web
repo = mainsail-crew/mainsail
path = ~/mainsail

#################################################################
All Configuration Files:
/home/pi/klipper_config/moonraker.conf
#################################################################
Unsafe Shutdown Count: 172

System Info:

***python***
  version: sys.version_info(major=3, minor=7, micro=3, releaselevel='final', serial=0)
  version_string: 3.7.3 (default, Jan 22 2021, 20:04:44)  [GCC 8.3.0]

***cpu_info***
  cpu_count: 4
  bits: 32bit
  processor: armv7l
  cpu_desc: ARMv7 Processor rev 3 (v7l)
  serial_number: 100000009664357c
  hardware_desc: BCM2711
  model: Raspberry Pi 4 Model B Rev 1.2
  total_memory: 3878088
  memory_units: kB

***sd_info***
  manufacturer_id: 03
  manufacturer: Sandisk
  oem_id: 5344
  product_name: SC16G
  product_revision: 8.0
  serial_number: 6cb44236
  manufacturer_date: 10/2017
  capacity: 14.8 GiB
  total_bytes: 15931539456

***distribution***
  name: Raspbian GNU/Linux 10 (buster)
  id: raspbian
  version: 10
  version_parts: {'major': '10', 'minor': '', 'build_number': ''}
  like: debian
  codename: buster
  release_info: {'name': 'MainsailOS', 'version_id': '0.6.1', 'codename': 'buster', 'id': 'mainsailos'}

***virtualization***
  virt_type: none
  virt_identifier: none

***network***

***canbus***

Systemd unit moonraker.service:
unit_name: moonraker.service
is_default: True
Properties:
**Requires=['network-online.target', '-.mount', 'sysinit.target', 'system.slice']
**After=['system.slice', 'sysinit.target', 'systemd-journald.socket', 'network-online.target', 'basic.target', '-.mount', 'klipper.service']
**SupplementaryGroups=['moonraker-admin']
**EnvironmentFiles=/home/pi/printer_data/systemd/moonraker.env
**ExecStart=/home/pi/moonraker-env/bin/python $MOONRAKER_ARGS
**WorkingDirectory=/home/pi/moonraker
**FragmentPath=/etc/systemd/system/moonraker.service
**Description=API Server for Klipper SV1
**User=pi

Systemd unit klipper.service:
unit_name: klipper.service
is_default: True
Properties:
**Description=Starts Klipper and provides klippy Unix Domain Socket API
**ExecStart=/home/pi/klippy-env/bin/python /home/pi/klipper/klippy/klippy.py ${KLIPPER_CONFIG} -l ${KLIPPER_LOG} -a ${KLIPPER_SOCKET}
**FragmentPath=/etc/systemd/system/klipper.service
Klipper Version: v0.10.0-631-g8a065287
2022-11-17 00:24:14,318 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: 127.0.0.1
2022-11-17 00:24:14,318 [authorization.py:_prune_conn_handler()] - Trusted Connection Expired, IP: fe80::1c59:915c:2c8f:1afb
2022-11-17 00:45:59,781 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: fe80::1c59:915c:2c8f:1afb
2022-11-17 00:45:59,784 [app.py:log_request()] - 101 GET /websocket (fe80::1c59:915c:2c8f:1afb) [_TRUSTED_USER_] 7.19ms
2022-11-17 00:45:59,785 [websockets.py:open()] - Websocket Opened: ID: 2823366384, Proxied: True, User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:107.0) Gecko/20100101 Firefox/107.0, Host Name: vzbot.local
2022-11-17 00:45:59,793 [websockets.py:_handle_identify()] - Websocket 2823366384 Client Identified - Name: mainsail, Version: 2.4.0, Type: web
2022-11-17 00:49:35,915 [app.py:log_request()] - 201 POST /server/files/upload (fe80::1c59:915c:2c8f:1afb) [_TRUSTED_USER_] 11.30ms
2022-11-17 00:50:13,347 [klippy_connection.py:_on_connection_closed()] - Klippy Connection Removed
2022-11-17 00:50:13,427 [websockets.py:on_close()] - Websocket Closed: ID: 2824681232 Close Code: 1000, Close Reason: None, Pong Time Elapsed: 0.37
2022-11-17 00:50:13,458 [authorization.py:_check_trusted_connection()] - Trusted Connection Detected, IP: 127.0.0.1
2022-11-17 00:50:13,481 [app.py:log_request()] - 101 GET /websocket?token=6463YVDKNIQBE7Z5I3E3LWWWGOQF2JB4 (127.0.0.1) [_TRUSTED_USER_] 2.77ms
2022-11-17 00:50:13,482 [websockets.py:open()] - Websocket Opened: ID: 2823366128, Proxied: False, User Agent: , Host Name: 127.0.0.1
2022-11-17 00:50:13,513 [websockets.py:on_close()] - Websocket Closed: ID: 2823366128 Close Code: None, Close Reason: None, Pong Time Elapsed: 0.03
2022-11-17 00:50:14,608 [klippy_connection.py:_do_connect()] - Klippy Connection Established
2022-11-17 00:50:15,389 [klippy_connection.py:_init_klippy_connection()] - Webhooks Subscribed
2022-11-17 00:50:15,394 [klippy_connection.py:_init_klippy_connection()] - GCode Output Subscribed
2022-11-17 00:50:16,893 [job_state.py:_handle_started()] - Job state initialized: standby
2022-11-17 00:50:17,158 [klippy_connection.py:_check_ready()] - Klippy ready
2022-11-17 00:50:17,655 [data_store.py:_init_sensors()] - Configuring available sensors: ['heater_bed', 'temperature_sensor zasilacz', 'extruder']
2022-11-17 00:50:17,691 [app.py:log_request()] - 101 GET /websocket?token=AQT6PBX7CGILRUQVO55AM34QJSPPAT4R (127.0.0.1) [_TRUSTED_USER_] 1.73ms
2022-11-17 00:50:17,692 [websockets.py:open()] - Websocket Opened: ID: 2823674992, Proxied: False, User Agent: , Host Name: 127.0.0.1
2022-11-17 00:50:17,727 [app.py:log_request()] - 404 GET /machine/device_power/devices (127.0.0.1) [No User] 1.74ms
2022-11-17 00:50:18,540 [file_manager.py:get_file_list()] - Updating File List <gcodes>...
2022-11-17 00:50:18,591 [app.py:log_request()] - 404 GET /machine/device_power/devices (127.0.0.1) [No User] 1.72ms
2022-11-17 00:50:19,271 [file_manager.py:get_file_list()] - Updating File List <gcodes>...
2022-11-17 01:18:40,258 [websockets.py:on_close()] - Websocket Closed: ID: 2823366384 Close Code: 1001, Close Reason: None, Pong Time Elapsed: 0.47
